pkgs/tools/backup/tsm-client/default.nix

```
@@ -1,15 +1,17 @@
 { lib
 , stdenv
+, fetchurl
 , autoPatchelfHook
+, openssl
+, zlib
 , buildEnv
-, fetchurl
 , makeWrapper
 , procps
-, zlib
+, gnugrep
 # optional packages that enable certain features
+, lvm2 ? null  # LVM image backup and restore functions
 , acl ? null  # EXT2/EXT3/XFS ACL support
 , jdk8 ? null  # Java GUI
```

```suggestion
, lvm2 ? null  # LVM image backup and restore functions
, acl ? null  # EXT2/EXT3/XFS ACL support
, jdk8 ? null  # Java GUI
```
those are never null and overriding something with null is discouraged.

---

In the spirit of other packages with variable configuration features, I added an option `enableGui` that switches the Java GUI or or off.  The `null` default values are dropped, so the `jdk8` variable is simply ignored if `enableGui=false`.